Dockerfile

```diff
@@ -145,7 +146,14 @@ RUN add-apt-repository ppa:ondrej/php \
     php7.4-tidy \
     php7.4-xml \
     php7.4-zip \
-    php7.4-zmq
+    php7.4-zmq \
+  # Perform a fake `composer update` to warm ~dependabot/.composer/cache with historic data
+  && mkdir /tmp/composer-cache \
+  && cd /tmp/composer-cache \
+  && echo '{"require":{"psr/log": "^1.1.3"}}' > composer.json \
+  && composer update --no-scripts --dry-run \
+  && cd /tmp \
+  && rm -Rf /tmp/composer-cache
```

Comment:
This might be a difficult ask but is it possible to see the size of this Docker layer before and after this change?

If it's possible to run `dive` locally to inspect the size of that layer that would be great for me.

```bash
モ dive --help
This tool provides a way to discover and explore the contents of a docker image. Additionally the tool estimates
the amount of wasted space and identifies the offending files from the image.

Usage:
  dive [IMAGE] [flags]
```

E.g.

Running:

```bash
モ dive mokhan/minbox
```

Produces:

![Screen Shot 2021-04-13 at 10 28 53](https://user-images.githubusercontent.com/80475/114587620-15f43580-9c43-11eb-8195-3196712d5d32.png)

I'm curious to know what the baseline is. Also, feel free to ignore this request if it's too much effort.

Reply:
@xlgmokha could you just run this locally for now? We've got the tracking issue to add it to CI.

The "before" image is `dependabot/dependabot-core:latest`, CI pushes this "after" image to `dependabot/dependabot-core:branch--refs-pull-3485-merge`:

<img width="2270" alt="Screen Shot 2021-04-14 at 7 25 23 AM" src="https://user-images.githubusercontent.com/1559510/114702852-96be3a80-9cf2-11eb-9440-f456b54e07c9.png">

Reply:
Looks like post change has slightly less wasted space, 70 vs 98mb, `dive` on current branch:

<img width="1607" alt="Screenshot 2021-04-14 at 11 46 15" src="https://user-images.githubusercontent.com/20165/114711498-efa7c600-9d26-11eb-9d4a-183b0495565f.png">

`dive` on `main`:

<img width="1626" alt="Screenshot 2021-04-14 at 13 38 59" src="https://user-images.githubusercontent.com/20165/114711558-fe8e7880-9d26-11eb-8aa3-cdbd4ea03902.png">

Reply:
Those elixir layers are huge, would be cool to do a push on those at some point, for another day 🙂